### PR TITLE
Add option for Sputnik V

### LINKED ISSUE #181: Add Sputnik support also

### PR DESCRIPTION
**Pull request by BumpyKnuckls**
Closes #181

### CODE REVIEW

File: src/utils.py [hunk 1]
```diff
@@ -618,14 +618,16 @@ def get_vaccine_preference():
         "It seems you're trying to find a slot for your first dose. Do you have a vaccine preference?"
     )
     preference = input(
-        "Enter 0 for No Preference, 1 for COVISHIELD, or 2 for COVAXIN. Default 0 : "
+        "Enter 0 for No Preference, 1 for COVISHIELD, 2 for COVAXIN or 3 for SPUTNIK V. Default 0 : "
     )
     preference = int(preference) if preference and int(preference) in [0, 1, 2] else 0
```
Comment (shailesh) [inline src/utils.py:623]: Add a check here [0, 1, 2, 3] for Sputnik validation

**Review by shailesh — CHANGES_REQUESTED** [review]
add a check in preferences for Sputnik validation

File: src/utils.py [hunk 2]
```diff
@@ -618,14 +618,16 @@ def get_vaccine_preference():
         "It seems you're trying to find a slot for your first dose. Do you have a vaccine preference?"
     )
     preference = input(
-        "Enter 0 for No Preference, 1 for COVISHIELD, or 2 for COVAXIN. Default 0 : "
+        "Enter 0 for No Preference, 1 for COVISHIELD, 2 for COVAXIN or 3 for SPUTNIK V. Default 0 : "
```
Comment (shailesh) [inline src/utils.py:621]: comma missing

File: src/utils.py (hunk 1 above)
Comment (BumpyKnuckls) [reply src/utils.py]: Thanks for the catch. Updated

**Comment by bombardier-gif** [comment]
Thanks a ton @BumpyKnuckls and @shailesh for the review!!
💯

**Review by shailesh — APPROVED** [review]
LGTM 👍

**Pull request merged**